presto-main/src/test/java/com/facebook/presto/operator/scalar/TestTDigestFunctions.java

```diff
@@ -452,6 +481,39 @@ public void testDestructureTDigest()
                 weights);
     }
 
+    @Test
+    public void testConstructTDigestLarge()
+    {
+        TDigest tDigest = createTDigest(STANDARD_COMPRESSION_FACTOR);
+        List<Double> values = new ArrayList<>();
+        for (int i = 0; i < NUMBER_OF_ENTRIES; i++) {
+            values.add((double) i);
+        }
+
+        values.stream().forEach(tDigest::add);
+
+        List<Integer> weights = Collections.nCopies(values.size(), 1);
```

Comment:
We might need a more realistic example (such as forming a small tdigest from a normal distribution with a high compression factor) but this is probably OK to start.

Reply:
Maybe it's also worth adding a test to ensure that `construct_tdigest(destructure_tdigest(tdigest)) = tdigest` and `destructure_tdigest(construct_tdigest(row)) = row` to show that they are indeed reciprocals?

Reply:
I'd just pick something high enough that the list of values that you get is still fairly readable.